src/kili/entrypoints/mutations/asset/__init__.py

```diff
@@ -140,14 +140,14 @@ def append_many_to_dataset(
         for key, value in field_mapping.items():
             if value is not None:
                 assets = [{**assets[i], key: value[i]} for i in range(nb_data)]
-        result = import_assets(
+        created_assets_ids = import_assets(
```

Comment:
so for async imports the asset_ids would be empty?  if yes, we should put it in the docstring